docs/usage.md

```diff
@@ -331,21 +354,31 @@ Skip the amplicon trimming step performed by Cutadapt. Use this if your input Fa
 
 By default, trimmed FastQ files will not be saved to the results directory. Specify this flag (or set to true in your config file) to copy these files to the results directory when complete (Default: false).
 
-## Alignments
+## Variant calling
+
+### `--callers`
+
+Specify which variant calling algorithms you would like to use. Available options are `varscan2`, `ivar` and `bcftools` (Default: 'varscan2,ivar,bcftools').
 
 ### `--ivar_exclude_reads`
 
 This option unsets the `-e` parameter in `ivar trim` to discard reads without primers (Default: false).
 
-### `--save_align_intermeds`
+### `--filter_dups`
 
-By default, intermediate [BAM](https://samtools.github.io/hts-specs/) files will not be saved. The final BAM files created after the appropriate filtering step are always saved to limit storage usage. Set to true to also save other intermediate BAM files (Default: false).
+Remove duplicate reads from alignments as identified by picard MarkDuplicates (Default: false). Note that unless you are using [UMIs](https://emea.illumina.com/science/sequencing-method-explorer/kits-and-arrays/umi.html) it is not possible to establish whether the fragments you have sequenced were derived via true biological duplication (i.e. sequencing independent template fragments) or as a result of PCR biases introduced during the library preparation.
 
-## Variant calling
+### `--min_base_qual`
 
-### `--callers`
+When performing variant calling skip bases with baseQ/BAQ smaller than this number (Default: 20).
 
-Specify which variant calling algorithms you would like to use. Available options are `varscan2` and `ivar` (Default: 'varscan2,ivar').
+### `--max_allele_freq`
```

Comment:
Mm, is it max_allele_frequency or min_allele_frequency? It's the minimum frequency needed for a call to be included in the consensus, right?

Reply:
Yeah, this can be confusing but in theory the `--min_allele_freq` would be what we use with the default parameters for any given variant caller...this parameter is specifically for the "scricter" filtering of the data. Happy for you to change it in another PR :)